clmm/utils.py

```diff
@@ -62,6 +62,7 @@ def make_bins(rmin, rmax, nbins=10, method='evenwidth'):
         Binning method to use
         'evenwidth' - Default, evenly spaced bins between rmin and rmax
         'evenlog10width' - Logspaced bins with even width in log10 between rmin and rmax
+        'equaloccupation' - Bins with equal occupation numbers
```

Comment:
Need to add new Parameter description:
`source_seps : array like
      radial distance of source separations`
(or some other wording for the docstrings)

Reply:
Thanks, done.